Gruntfile.coffee

```diff
@@ -1230,10 +1230,10 @@ module.exports = (grunt) ->
 				files: "site/pages/docs/**/*.hbs"
 				tasks: "pages:docs"
 
+
 		eslint:
 			options:
-				configFile: '.eslintrc.json'
-				quiet: true
```

Comment:
The new warnings you get are because of this that I accidentally knocked out in my example